components/script/dom/htmlformelement.rs

```diff
@@ -97,7 +96,9 @@ pub(crate) struct HTMLFormElement {
     controls: DomRefCell<Vec<Dom<Element>>>,
 
     #[allow(clippy::type_complexity)]
-    past_names_map: DomRefCell<HashMapTracedValues<Atom, (Dom<Element>, NoTrace<Instant>)>>,
+    past_names_map: DomRefCell<HashMapTracedValues<Atom, (Dom<Element>, NoTrace<usize>)>>,
+    // the current generation of past names, i.e., the number of name changes
+    past_names_generation: Cell<usize>,
```

Comment:
rustdoc comments should begin with three slashes and be complete sentences:

```suggestion

    /// The current generation of past names, i.e., the number of name changes to the name.
    current_name_generation: Cell<usize>,
```